solutions/community-deploy/cloud/AWS/DEPLOY.md

```diff
@@ -0,0 +1,80 @@
+# Setup K8s Cluster and Deploy CORTX Stack
+
+   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+
+## Prerequisite 
+### Minimum specification required for Cortx Stack
+   - RAM: 16GB
+   - CPU: 8Core
+   - DISK: Need 8 Disk with 10GB per disk.
+
+### If you have automated deployment for solution yaml, please make sure your vm having below drives available
+```
+      ls /dev/sd*
+```
+   - `/dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi  /dev/sdj`
+
+### If Git doesn't exist in your system then run below command to install it
+
+```
+      yum install git -y
+```
+### SELinux should be disable
+   - Use below command to check status of SELinux.
+```
+      sestatus
+```
+   - If SELinux is enable then you can run below command to disabled SELinux.
+
+```
+      sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
+```
+
+   - Once above command ran successfully then reboot your system.
+
+```
+      reboot
+```   
+### Note: 
+ 1. All node should be ssh connection happen properly
+ 2. If you wanted to deploy on AWS environment then follow this document [link](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md)
```

Comment:
Please change this to 
`You also deploy CORTX on AWS EC2 instance. Please follow CORTX Deployment on AWS <Add symlink to this>`

Reply:
Changed above line